### ISSUE TITLE
 Official Website of Teamengine error!

### ISSUE THREAD

**Issue by jesily** [opengeospatial/teamengine#423]
After logged into the teamengine page(https://cite.opengeospatial.org/teamengine/), get the error message: 

HTTP Status 500 - javax.servlet.ServletException: org.xml.sax.SAXParseException; Premature end of file. 

type Exception report

message javax.servlet.ServletException: org.xml.sax.SAXParseException; Premature end of file.
description The server encountered an internal error that prevented it from fulfilling this request.
exception ...

**Comment by dstenger** [comment]
Thank you for reporting.

I am able to log in with my personal account.
Can you please re-check if the problem is still occurring as we had some maintenance work yesterday?

**Comment by dstenger** [comment]
Following exception is thrown:
```
[Fatal Error] :-1:-1: Premature end of file.
Feb 14, 2020 9:30:54 AM org.apache.catalina.core.StandardWrapperValve invoke
SEVERE: Servlet.service() for servlet [jsp] in context with path [/teamengine] threw exception [javax.servlet.ServletException: org.xml.sax.SAXParseException; Premature end of file.] with root cause
org.xml.sax.SAXParseException; Premature end of file.
	at org.apache.xerces.parsers.DOMParser.parse(Unknown Source)
	at org.apache.xerces.jaxp.DocumentBuilderImpl.parse(Unknown Source)
	at javax.xml.parsers.DocumentBuilder.parse(DocumentBuilder.java:205)
	at com.occamlab.te.web.TestSession.load(TestSession.java:95)
	at org.apache.jsp.viewSessions_jsp._jspService(viewSessions_jsp.java:199)
	at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:70)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:731)
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:439)
	at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:395)
	at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:339)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:731)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:303)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:220)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:122)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:614)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:170)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:103)
	at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:957)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:116)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:423)
	at org.apache.coyote.ajp.AjpProcessor.process(AjpProcessor.java:190)
	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:620)
	at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:316)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
```

**Comment by dstenger** [comment]
@keshav-nangare Can you please take a look at this problem?

**Comment by dstenger** [comment]
In this case, session.xml is empty.
@keshav-nangare Can you please create a fix ignoring empty session files instead of throwing an exception.

**Comment by keshav-nangare** [comment]
Fixed with #425 PR.
